src/registrar/templates/base.html

```diff
@@ -103,6 +103,20 @@
       </div>
     </div>
   </section>
+  {% if IS_DEMO_SITE %}
+    <section
+    class="usa-site-alert usa-site-alert--emergency usa-site-alert--no-icon"
+    aria-label="Site alert,,,,,"
```

Comment:
Lots and lots of `,`'s?

Reply:
Copied and pasted from https://designsystem.digital.gov/components/site-alert/ but now I see they were using the commas to differentiate between many different components with the same label. Fixed.